tests/unit_tests/handlers/test_delivery_organise_handlers.py

```diff
@@ -0,0 +1,163 @@
+from tornado.testing import *
+from tornado.web import Application
+
+import tempfile
+import mock
+import pathlib
+
+from delivery.app import routes
+
+from tests.test_utils import DummyConfig
+
+class TestOrganiseHandlers(AsyncHTTPTestCase):
+    API_BASE = "/api/1.0"
+
+    def get_app(self):
```

Comment:
is the `get_app` function called automatically by the test framework when setting up the tests?